### PR TITLE
Removed API version validation.

### PR DESCRIPTION
**Pull request by filipe-kovalski**
It is now possible to use API 1.2 on OneView 2.0

### CODE REVIEW

File: oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java [hunk 1]
```diff
@@ -117,11 +116,7 @@ private void validateApiVersion(final int actualVersion, final int expectedVersi
             logger.info("API version : " + "You are trying to connect to appliance verion: " + actualVersion
```
Comment (lhsvobodaj) [inline oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java]: We could change the versions equality test to a less than or equal. Doing that we would still have a log entry message when the actual version is less than the expected version (which is a valid usage of the SDK).

Comment (dronin) [inline oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java]: The correct test would be :
applianceDTO.getMinimumVersion() <= requestedVersion <= applianceDTO.getCurrentVersion()

But we also need to make sure the requested version is supported by the library not just the appliance.  How do we handle a request for a API version that returns an object version not supported by the library?

Comment (filipe-kovalski) [inline oneview-sdk-java-lib/src/main/java/com/hp/ov/sdk/util/SdkUtils.java]: Thanks for pointing the minimum version =)

The SDK only supports API 1.2 for now.
If the user decides to use another API version, the result will probably be a parsing (JSON / Java) error or a message regarding a missing field from 2.0.

We will work on a solution for that.

Again, thanks!

**Comment by lhsvobodaj** [comment]
:+1:

**Pull request merged**